ibc-core/ics03-connection/src/handler/conn_open_ack.rs

```diff
@@ -112,7 +112,7 @@ where
                 prefix_on_b,
                 &msg.proof_client_state_of_a_on_b,
                 consensus_state_of_b_on_a.root(),
-                Path::ClientState(ClientStatePath::new(vars.client_id_on_b().clone())),
```

Comment:
Can we keep this `Path::*(*)` as it is? I like this as it is verbose and only for internal uses.

Reply:
[e5dfbfb](https://github.com/cosmos/ibc-rs/pull/1273/commits/e5dfbfb797312f46a65b019551deebf026634987)